practicesharp/Number.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    class Number
+    {
+        
+        public string number { get; private set; }
```

Comment:
"Number" class should describe a Number, but it has a property which is called number and its type is a string.
Pretty weird :/
There are 2 problems here, the first is that the name of the class does not describe well the purpose of the class, the second is that because you use the constructor to extract the number from the string, you have to create a property and contain a state.
A better name for the class can be "NumberFromStringExtractor".
The constructor should be a function that receives a string (without ref) and returns a number.